### PR TITLE
4 developers have a shared easily deployed docker environment for balancer backend

### PR DESCRIPTION
**Pull request by jusjayson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+DOCKER_CTX_FROM_PROJECT_ROOT ?= .
+DOCKER_CTX_FROM_PYTHON_DOCKER ?= ..
+DOCKER_ENTRYPOINT_DEST ?= /entrypoint.sh
+DOCKER_LOCAL_CMD ?= /bin/bash
+DOCKER_LOG_FOLDER_PATH ?= /root/.$(PROJECT_NAME)/logs
+DOCKER_NO_CACHE ?=  # Define with any value to disable caching on build
+DOCKER_PROJECT_ROOT_FROM_CTX ?= .
+DOCKER_PROJECT_SERVICE_NAME ?= balancer-backend
+DOCKER_PYTHON_DOCKER_FROM_CTX ?= Python-Docker
+DOCKER_REGISTRY ?= docker.fakecodeforphillydomain.com
```
Comment (jusjayson) [inline Makefile:28]: Is there a Code for Philly container registry we can use so that people don't need to build? We may want base env files as well.

**Comment by dherbst** [comment]
The only thing I had to do was run `DONT_PASS_SSH_KEYS=1 make build-project` to get it to work, but once I did that I was able to run `make launch-local-project` and it ran the docker container.  

LGTM

**Review by dherbst — APPROVED** [review]
had to run with `DONT_PASS_SSH_KEYS=1 make build-project` which works because the project is all public repos.

**Comment by jusjayson** [comment]
Thanks for finding that! I'll make an update to the makefile before
pushing, since (as you mentioned) repos are all public

**Comment by joshuagerstein** [comment]
When I try to run make in windows, I get the following error: 
```
if ! [ -d Python-Docker ]; then\
        git clone https://github.com/jusjayson/Python-Docker.git;\
fi
[ was unexpected at this time.
make: *** [Makefile:44: pull-python-docker] Error 255
```

I don't know makefile syntax well enough to propose a way to make it work without bash commands.

**Comment by jusjayson** [comment]
Possible Darrel's discovery will fix this. Will update and push tomorrow
morning

**Comment by jusjayson** [comment]
Can you also open an issue about checking windows compatibility? I need to
look into how windows handles make commands

**Comment by dherbst** [comment]
LGTM, merging.

**Pull request merged**